JaphethCA/More-recipe
# #Makes Search and Profile view Responsive

## What does this PR do?
- it makes search form and profile view component more responsive to mobile devices
- it renames components and server files to meet airbnb style 

## How can this PR be tested?
- clone repository
- install dependencies
- start local server with `npm run start:dev`
- navigate to [http://localhost:50000/](http://localhost:50000/)
- check search and profile view responsiveness
